staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/config.go

```diff
@@ -111,6 +111,7 @@ func init() {
 	utilruntime.Must(apiserverconfig.AddToScheme(configScheme))
 	utilruntime.Must(apiserverconfigv1.AddToScheme(configScheme))
 	codecs = serializer.NewCodecFactory(configScheme)
+	metrics.RegisterMetrics()
```

Comment:
You want all 3 metrics packages here to be safe:

```suggestion
	metrics.RegisterMetrics()
	storagevalue.RegisterMetrics()
	encryptionconfigmetrics.RegisterMetrics()
```

Also we probably need to backport this specific change (confirm that these metrics don't work on older releases before doing the backport).

Reply:
@ritazh opened #120543